network-config-analyzer/PeerContainer.py

```diff
@@ -40,44 +30,28 @@ def __init__(self, ns_resources=None, peer_resources=None, config_name='global')
         """
         self.peer_set = PeerSet()
         self.namespaces = {}  # mapping from namespace name to the actual K8sNamespace object
+        self.services = {}  # mapping from service name to the actual K8sService object
```

Comment:
service name is not necessarily unique, as we may have 2 different services with the same name but on a different namespace.

Reply:
Done